gtk4/sample/examples/application1/exampleapp.rb

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+exec ruby -x "$0" "$@"
+#!ruby
+
```

Comment:
How about simplifying this shebang?

```suggestion
#!/usr/bin/env ruby
```